ndbench-cockroachdb-plugins/src/main/java/com/netflix/ndbench/plugin/cockroachdb/configs/CockroachDBConfigurationBase.java

```diff
@@ -0,0 +1,43 @@
+/*
+ *  Copyright 2018 Netflix, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+package com.netflix.ndbench.plugin.cockroachdb.configs;
+
+import com.netflix.archaius.api.annotations.DefaultValue;
+import com.netflix.archaius.api.annotations.PropertyName;
+
+/**
+ * Configurations for Cockroachdb benchmarks
+ *
+ * @author Sumanth Pasupuleti
+ */
+public interface CockroachDBConfigurationBase {
+    @PropertyName(name = "dbname")
+    @DefaultValue("perftest")
+    String getDBName();
+
+    @PropertyName(name = "tablename")
+    @DefaultValue("test")
+    String getTableName();
+
+    @PropertyName(name = "loadbalancer")
+    @DefaultValue("cdecrdbnossl--useast-elb-1849424046.us-east-1.elb.amazonaws.com")
```

Comment:
Is it fine to publically expose your ELB endpoint as a property?

Reply:
Good point; removing it.